package.json

```diff
@@ -35,16 +35,21 @@
         "@types/node": "^12.7.9",
         "@types/sinon": "^7.5.0",
         "@types/sinon-chai": "^3.2.3",
+        "@typescript-eslint/eslint-plugin": "^2.3.3",
+        "@typescript-eslint/eslint-plugin-tslint": "^2.3.3",
+        "@typescript-eslint/parser": "^2.3.3",
         "chai": "^4.2.0",
         "del": "^5.1.0",
+        "eslint": "^6.5.1",
         "husky": "^3.0.8",
         "mocha": "^6.2.1",
         "prettier": "^1.18.2",
         "pretty-quick": "^1.11.1",
         "sinon": "^7.5.0",
         "sinon-chai": "^3.3.0",
-        "tslint": "5.20.0",
-        "tslint-config-prettier": "^1.18.0"
+        "tslint": "^5.20.0",
+        "tslint-config-prettier": "^1.18.0",
```

Comment:
These two tslint packages should be removed. I think that'll be possible once the `tslint/config` rules are no longer included?